test/controllers/users_controller_test.rb

```diff
@@ -219,7 +219,6 @@ class UsersControllerTest < ActionController::TestCase
     get :extlogin, session: {:user => users(:admin).id }
     assert_redirected_to hosts_path
     users(:admin).reload
-    assert users(:admin).last_login_on.to_i >= time.to_i, 'User last login time was not updated'
```

Comment:
this should definitelly stay, we want this to pass.

Reply:
we will probably need to keep the assign as well :)

Reply:
I think that's a unit test, the unit test of UserController. The call of User.post_successful_login has moved from UserController to User model. So there is no reason to have there this line of code because It's not in scope of controller to perform the update of last_login value. In this opinion the removed assertion should be removed. 

Maybe there is better place to test the update of value. Any ideas? @ezr-ondrej

Reply:
This is testing `extlogin` and we are updating the times directly there now, isn't that right? And we want this to happen once logged, so I see added value in testing it here :)